invenio/modules/jsonalchemy/scripts/jsonschema.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
```

Comment:
What is the magic that make `inveniomanage` to discover my new manage script in the former case?